sources/install/package_base.sh

```diff
@@ -286,9 +294,6 @@ function package_base() {
     # NVM (install in conctext)
     zsh -c "source ~/.zshrc && nvm install node"
 
-    # FZF
-    touch /usr/share/doc/fzf/examples/key-bindings.zsh
```

Comment:
It was useless ?

Reply:
Yes because via git everything is in the clone folder